### DISCUSSION TITLE
第2节课作业(1班)

### DISCUSSION THREAD

**Discussion by KMnO4-zx** [InternLM/Tutorial#113] [作业]
**基础作业：**

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
- 熟悉 hugging face 下载功能，使用 `huggingface_hub` python 包，下载 `InternLM-20B` 的 config.json 文件到本地（需截图下载过程）。

**进阶作业（可选做）**

- 完成浦语·灵笔的图文理解及创作部署（需截图）
- 完成 Lagent 工具调用 Demo 创作部署（需截图）

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/06/ai-internlm-useage/

**Comment by woxinyonghen** [comment]
https://github.com/woxinyonghen/InternLMCamp/tree/main/homework01/基础作业/
https://github.com/woxinyonghen/InternLMCamp/tree/main/homework01/进阶作业/

**Comment by Aruisir** [comment]
https://github.com/Aruisir/InternLM/tree/0cb9ec693ff678c590fd165e0893977407cd2c39/%E7%AC%AC%E4%BA%8C%E8%AF%BE

**Comment by wudemoai** [comment]
https://blog.csdn.net/weixin_48478550/article/details/135430892?spm=1001.2014.3001.5501

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135432443

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/homework/chap1-homework.md

**Comment by Gu-Xiao** [comment]
作业+笔记（作业区＋笔记区均已提交）  https://blog.csdn.net/xidierzuo/article/details/135434824

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/KnFwdgJYfoHpiQxiSV7cN5hNnsb?from=from_copylink

**Comment by lds-coder** [comment]
https://blog.csdn.net/m0_73199148/article/details/135440623?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135440623%22%2C%22source%22%3A%22m0_73199148%22%7D

**Comment by 10086zgyd** [comment]
https://github.com/10086zgyd/InternLM

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135433855?spm=1001.2014.3001.5502

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135440100
基础作业在2.6节和5.2.1 节，进阶作业在第4章（4.5小节）和第3章（3.4小节）

**Comment by gpt-dance** [comment]
https://zhuanlan.zhihu.com/p/676513982

**Comment by Marxlp** [comment]
https://github.com/Marxlp/internLM-course-notes/blob/main/lecture1_homework.md

**Comment by Xpg74138** [comment]
https://github.com/Xpg74138/InternLMCamp/blob/main/First_homework.ipynb

**Comment by Andy1314Chen** [comment]
https://www.bilibili.com/read/cv29181045/?jump_opus=1

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/377bb3c3463f402783813869a98df4b2  作业截图在末尾

**Comment by random-zhou** [comment]
http://t.csdnimg.cn/kgdI6

**Comment by RongjianLiang** [comment]
https://zhuanlan.zhihu.com/p/676752016

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/676742628

**Comment by formorebeat** [comment]
https://shimo.im/docs/1d3aMxv7QdfM6l3g/

**Comment by woml** [comment]
https://woml.github.io/homework_2

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/676945852?

**Comment by strong-leaf** [comment]
大模型(书生.浦语大模型)finetune实战_lesson2笔记与作业 - warm green的文章 - 知乎
https://zhuanlan.zhihu.com/p/677003853

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135440530  
![Uploading image.png…]()
